migrations/0037_backfill_reblogs_count_and_add_in_reply_to_id_index.sql

```
@@ -0,0 +1,7 @@
+UPDATE objects SET reblogs_count = (
+  SELECT COUNT(*) FROM actor_reblogs
+  WHERE actor_reblogs.object_id = objects.id
+);
+
+CREATE INDEX IF NOT EXISTS "objects_in_reply_to_id" ON "objects" ("in_reply_to_id")
+  WHERE "in_reply_to_id" IS NOT NULL;
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)